platform/security/platform-security-core-api/src/main/java/ddf/security/PropertiesFileReader.java

```diff
@@ -11,7 +11,7 @@
  * is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.ui.admin.api.util;
+package ddf.security;
```

Comment:
I think the root issue is that `ddf.security` also contains `PropertiesLoader` and it wouldn't make sense to: 
* Disburse our property loading logic
* Have `platform-util` depend on `security-core-api` 

_Update_: After talking with @stustison we will depreciate `PropertiesLoader` since it is in an api bundle, have it instead invoke a version of itself in platform-util, and move all property logic there appropriately.